test/state/host.cpp

```diff
@@ -323,7 +324,11 @@ bytes32 Host::get_block_hash(int64_t block_number) const noexcept
     (void)block_number;
     // TODO: This is not properly implemented, but only single state test requires BLOCKHASH
     //       and is fine with any value.
-    return {};
+
```

Comment:
Is it temporary change until block hash PR will be merged. Right?

Reply:
Seems we still need the fallback "fake" mode for raw state tests (used by goevmlab). The `t8n` should receive block hashes in `"blockHashes"`. @holiman ? @winsvega ?

Reply:
Moved to #698.